TriblerGUI/tribler_request_manager.py

```diff
@@ -288,7 +288,7 @@ def on_finished(self, reply, capture_errors):
 
         data = reply.readAll()
         try:
-            json_result = json.loads(str(data), encoding='latin_1')
+            json_result = json.loads(str(data), encoding='utf-8')
```

Comment:
We also send non-utf-8 encoded strings over the request manager.

Reply:
Request manager still uses `latin_1` now and the config is converted to use `UTF-8`.